test/run_test.py

```diff
@@ -140,8 +154,16 @@ def run_test(executable, test_module, test_directory, options, *extra_unittest_a
     # Can't call `python -m unittest test_*` here because it doesn't run code
     # in `if __name__ == '__main__': `. So call `python test_*.py` instead.
     argv = [test_module + '.py'] + unittest_args + list(extra_unittest_args)
+
+    # Add 'test_directory' to PYTHONPATH so that imports work.
+    env = os.environ.copy()
+    if 'PYTHONPATH' in env:
+        env['PYTHONPATH'] = os.pathsep.join([env['PYTHONPATH'], test_directory])
+    else:
+        env['PYTHONPATH'] = test_directory
```

Comment:
I thought the point of all the discussing we did was so that you don't have to do these lines